src/common/context/forms/index.tsx

```diff
@@ -28,25 +30,32 @@ export const useFormsContext = (): FormsContext => useContext<FormsContext>(Form
 export const FormsContextProvider: FunctionComponent = ({ children }) => {
   const [activeFormIndex, setActiveFormIndex] = useState<number | undefined>(undefined);
   const [forms, setForms] = useState<FormEntry[]>([]);
+  const { config } = useConfigContext();
+
+  const currentForm = activeFormIndex === undefined ? undefined : forms[activeFormIndex];
+  const currentFormData = currentForm ? currentForm.data : undefined;
```

Comment:
what version of TS you use ?

can eventually write it like `const currentFormData = currentForm?.data`